pino.js

```diff
@@ -48,6 +48,7 @@ const hostname = os.hostname()
 const defaultErrorSerializer = stdSerializers.err
 const defaultOptions = {
   level: 'info',
+  levels,
```

Comment:
To avoid a circular dependency, I couldn't require the levels in tools.js. I added the default level here instead of adding a copy. This object is not exported and is only used by `createArgsNormalizer`. I'm not sure if there's a better solution?